features/step_definitions/collection_steps.rb

```diff
@@ -42,15 +42,17 @@
   step %{I post the work}
 end
 
-When(/^I view the(?: ([^"]*)) collection items page for "(.*?)"$/) do |item_status, collection|
+When /^I view the(?: ([^"]*)) collection items page for "(.*?)"$/ do |item_status, collection|
```

Comment:
If you're eliminating the default `else` option, the first param here should no longer be optional.